alembic/versions/0193b4725f32_bug_1132858_remove_extensions.py

```diff
@@ -0,0 +1,51 @@
+"""bug 1132858 remove extensions table
+
+Revision ID: 0193b4725f32
+Revises: bb8cdbb8a6bd
+Create Date: 2018-01-31 14:21:41.032179
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '0193b4725f32'
+down_revision = 'bb8cdbb8a6bd'
+
+from alembic import op
+from socorro.lib import citexttype, jsontype, buildtype
+from socorro.lib.migrations import fix_permissions, load_stored_proc
+
+import sqlalchemy as sa
+from sqlalchemy import types
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql import table, column
+
+
+def upgrade():
+    # Get a list of ALL tables that start with 'extensions_*' and drop them
+    connection = op.get_bind()
+    cursor = connection.connection.cursor()
+    cursor.execute("""
+        SELECT c.relname FROM pg_catalog.pg_class c
+        WHERE c.relkind = 'r' AND c.relname LIKE 'extensions_%'
```

Comment:
Is this the same as searching through `information_schema.tables`? My Postgres-fu is very weak.

Reply:
My postgres-fu is also weak.

This isn't my style of SQL--I don't tend to do single-letter names. I pulled this code from a previous migration and kind of left it as is because it worked before and it didn't seem super helpful to hone it into something better.

I can hone if you think it helps, though. It probably wouldn't take long and it's easily tested.